app/utils/mail/mailworker.py

```diff
@@ -1,13 +1,22 @@
 import smtplib
 import ssl
 from email.message import EmailMessage
+from pathlib import Path
 from typing import TYPE_CHECKING
 
 if TYPE_CHECKING:
     from app.core.config import Settings
 
 
-def send_email(recipient: str, subject: str, content: str, settings: "Settings"):
+def send_email(
+    recipient: str | list[str],
+    subject: str,
+    content: str,
+    settings: "Settings",
+    file_path: str | None = None,
```

Comment:
Ask for folder name and file uuid, then use file manipulation utils, to prevent access to files outside `/data`